bokeh/charts/chart.py

```diff
@@ -82,14 +82,14 @@ class Chart(Plot):
     def __init__(self):
 
         # Initializes then gets default properties
-        super(Chart, self).__init__(id=self._options.id or make_id())
+        # NOTE: we use option._id here since ChartOptions converts id arg to _id
+        super(Chart, self).__init__(id=self._options._id or make_id())
```

Comment:
Ok.. so thinking more about this I'd propose to just make the quick fix go into this PR so charts works with client sessions and then open an issue (where we can actually host a discussion) with a following PR to remove this and other leftovers.. how does it sounds?